### ISSUE TITLE
npm version gives (key + ":").padEnd

### ISSUE THREAD

**Issue by wgroenewold** [commitizen/cz-cli#744]
Yarn version works as expected.

**Comment by dmwelch** [comment]
```
$ npm version
{
  commitizen: '0.0.0-semantically-released',
  npm: '6.12.1',
  ares: '1.15.0',
  brotli: '1.0.7',
  cldr: '35.1',
  http_parser: '2.8.0',
  icu: '64.2',
  llhttp: '1.1.4',
  modules: '72',
  napi: '5',
  nghttp2: '1.39.2',
  node: '12.13.1',
  openssl: '1.1.1d',
  tz: '2019c',
  unicode: '12.1',
  uv: '1.33.1',
  v8: '7.7.299.13-node.16',
  zlib: '1.2.11'
}
```

I can't reproduce this.  Can you provide some more information on your setup? 
OS: 
Output of `npm --version`:
Output of `nvm --version`:
Anything else that you think might be relevant.

**Comment by wgroenewold** [comment]
npm --version > 6.14.4
nvm --version > zsh: command not found: nvm

Maybe I upgraded npm in the time between...

**Comment by dmwelch** [comment]
Can you paste the output of `npm version`?  Any error messages or debug info would help narrow down what is wrong.

**Comment by wgroenewold** [comment]
```
{
  npm: '6.14.4',
  ares: '1.16.0',
  brotli: '1.0.7',
  cldr: '37.0',
  http_parser: '2.9.3',
  icu: '67.1',
  llhttp: '2.0.4',
  modules: '72',
  napi: '6',
  nghttp2: '1.41.0',
  node: '12.18.0',
  openssl: '1.1.1g',
  tz: '2019c',
  unicode: '13.0',
  uv: '1.37.0',
  v8: '7.8.279.23-node.37',
  zlib: '1.2.11'
}


   ╭────────────────────────────────────────────────────────────────╮
   │                                                                │
   │      New patch version of npm available! 6.14.4 → 6.14.7       │
   │   Changelog: https://github.com/npm/cli/releases/tag/v6.14.7   │
   │               Run npm install -g npm to update!                │
   │                                                                │
   ╰────────────────────────────────────────────────────────────────╯
```

**Comment by dmwelch** [comment]
Huh?  What's the output of `npm install --dry-run -g commitizen --verbose`?  If that ends with something like:
```
+ commitizen@4.2.1
added 217 packages in 13.437s
npm verb exit [ 0, true ]
npm timing npm Completed in 13727ms
npm info ok 
```
can you run the command without the `-g` flag in a git-managed directory and paste the output?

**Comment by wgroenewold** [comment]
Without -g:

```
npm info it worked if it ends with ok
npm verb cli [
npm verb cli   '/usr/local/bin/node',
npm verb cli   '/usr/local/bin/npm',
npm verb cli   'install',
npm verb cli   '--dry-run',
npm verb cli   'commitizen',
npm verb cli   '--verbose'
npm verb cli ]
npm info using npm@6.14.4
npm info using node@v12.18.0
npm verb npm-session 4783a1109d445f17
npm http fetch GET 200 https://registry.npmjs.org/commitizen 16ms (from cache)
npm timing stage:loadCurrentTree Completed in 52ms
npm timing stage:loadIdealTree:cloneCurrentTree Completed in 0ms
npm timing stage:loadIdealTree:loadShrinkwrap Completed in 2ms
npm http fetch GET 304 https://registry.npmjs.org/find-node-modules 192ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/cz-conventional-changelog 202ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/dedent 204ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/glob 197ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/find-root 202ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/cachedir 212ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/is-utf8 200ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/fs-extra 216ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/detect-indent 226ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/inquirer 220ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/lodash 47ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/minimist 50ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/strip-json-comments 51ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/strip-bom 53ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/longest 51ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/word-wrap 55ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/lodash.map 58ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/chalk 61ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/conventional-commit-types 67ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/@commitlint%2fload 69ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/resolve-from 40ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/cosmiconfig 59ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/@commitlint%2ftypes 63ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/@commitlint%2fresolve-extends 67ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/@commitlint%2fexecute-rule 74ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/resolve-global 30ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/import-fresh 33ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/parent-module 36ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/callsites 29ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/global-dirs 25ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/ini 33ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/ansi-styles 33ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/supports-color 35ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/@types%2fcolor-name 38ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/color-convert 40ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/color-name 30ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/has-flag 23ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/yaml 37ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/parse-json 43ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/path-type 47ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/@types%2fparse-json 50ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/error-ex 35ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/json-parse-even-better-errors 39ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/lines-and-columns 39ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/@babel%2fcode-frame 49ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/@babel%2fhighlight 28ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/js-tokens 27ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/@babel%2fhelper-validator-identifier 46ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/escape-string-regexp 38ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/is-arrayish 32ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/merge 24ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/findup-sync 34ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/detect-file 37ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/is-glob 38ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/micromatch 46ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/resolve-dir 51ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/is-extglob 25ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/array-unique 64ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/arr-diff 66ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/define-property 70ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/braces 73ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/nanomatch 72ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/kind-of 74ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/object.pick 78ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/extend-shallow 86ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/extglob 86ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/fragment-cache 91ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/regex-not 46ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/to-regex 38ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/snapdragon 47ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/isobject 31ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/arr-flatten 39ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/repeat-element 40ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/fill-range 47ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/snapdragon-node 48ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/split-string 49ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/is-extendable 21ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/is-number 29ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/to-regex-range 30ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/repeat-string 34ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/is-buffer 24ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/base 44ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/debug 44ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/source-map-resolve 44ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/use 45ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/source-map 47ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/map-cache 52ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/cache-base 38ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/pascalcase 37ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/class-utils 42ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/component-emitter 43ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/mixin-deep 43ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/collection-visit 40ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/to-object-path 49ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/set-value 49ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/has-value 51ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/union-value 52ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/get-value 58ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/unset-value 57ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/object-visit 28ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/map-visit 32ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/has-values 27ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/is-plain-object 23ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/assign-symbols 28ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/arr-union 22ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/isarray 24ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/static-extend 28ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/is-descriptor 23ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/is-accessor-descriptor 30ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/is-data-descriptor 31ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/object-copy 22ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/copy-descriptor 33ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/for-in 27ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/ms 25ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/urix 46ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/source-map-url 47ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/resolve-url 49ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/decode-uri-component 50ms (from cache)
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm http fetch GET 304 https://registry.npmjs.org/atob 51ms (from cache)
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm http fetch GET 304 https://registry.npmjs.org/snapdragon-util 43ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/safe-regex 22ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/ret 24ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/expand-brackets 24ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/posix-character-classes 26ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/is-windows 22ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/expand-tilde 25ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/global-modules 28ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/homedir-polyfill 26ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/parse-passwd 23ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/global-prefix 31ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/which 26ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/isexe 30ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/graceful-fs 26ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/universalify 29ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/jsonfile 32ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/fs.realpath 42ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/minimatch 43ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/path-is-absolute 43ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/once 48ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/inflight 51ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/inherits 54ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/wrappy 25ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/brace-expansion 27ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/concat-map 24ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/balanced-match 26ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/cli-width 55ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/cli-cursor 59ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/external-editor 60ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/string-width 59ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/run-async 63ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/strip-ansi 64ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/ansi-escapes 70ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/mute-stream 71ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/rxjs 73ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/figures 86ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/through 36ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/restore-cursor 33ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/signal-exit 5ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/onetime 26ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/mimic-fn 29ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/chardet 6ms (from cache)
npm http fetch GET 200 https://registry.npmjs.org/tmp 6ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/iconv-lite 25ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/safer-buffer 21ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/os-tmpdir 22ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/tslib 25ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/is-fullwidth-code-point 25ms (from cache)
npm http fetch GET 304 https://registry.npmjs.org/ansi-regex 22ms (from cache)
npm timing stage:loadIdealTree:loadAllDepsIntoIdealTree Completed in 3043ms
npm timing stage:loadIdealTree Completed in 3111ms
npm timing stage:generateActionsToTake Completed in 50ms
npm WARN enoent ENOENT: no such file or directory, open '/Users/wouter/vagrant-local/package.json'
npm verb enoent This is related to npm not being able to find a file.
npm verb enoent
npm WARN vagrant-local No description
npm WARN vagrant-local No repository field.
npm WARN vagrant-local No README data
npm WARN vagrant-local No license field.

npm timing audit submit Completed in 256ms
npm http fetch POST 200 https://registry.npmjs.org/-/npm/v1/security/audits/quick 256ms
npm timing audit body Completed in 1ms
+ commitizen@4.2.1
added 217 packages and audited 217 packages in 3.538s
found 0 vulnerabilities

npm verb exit [ 0, true ]
npm timing npm Completed in 4029ms
npm info ok


   ╭────────────────────────────────────────────────────────────────╮
   │                                                                │
   │      New patch version of npm available! 6.14.4 → 6.14.8       │
   │   Changelog: https://github.com/npm/cli/releases/tag/v6.14.8   │
   │               Run npm install -g npm to update!                │
   │                                                                │
   ╰────────────────────────────────────────────────────────────────╯
```

**Comment by dmwelch** [comment]
Looks like it works now. 🥳   Not sure what was up but I suspect something in your setup was off - reopen the ticket if you re-encounter the issue.  Cheers!
